docs/releases/v3.3.0.md

```diff
@@ -2,43 +2,51 @@
+| `alertmanager`        | [`v0.26.0`](https://github.com/prometheus/alertmanager/releases/tag/v0.26.0)                               | No Update                    |
+| `blackbox-exporter`   | [`v0.25.0`](https://github.com/prometheus/blackbox_exporter/releases/tag/v0.25.0)                          | v0.24.0                      |
+| `grafana`             | [`v11.3.0`](https://github.com/grafana/grafana/releases/tag/v11.3.0)                                       | v9.5.5                       |
+| `kube-rbac-proxy`     | [`v0.18.1`](https://github.com/brancz/kube-rbac-proxy/releases/tag/v0.18.1)                                | v0.14.0                      |
+| `kube-state-metrics`  | [`v2.13.0`](https://github.com/kubernetes/kube-state-metrics/releases/tag/v2.13.0)                         | v2.9.2                       |
+| `node-exporter`       | [`v1.8.2`](https://github.com/prometheus/node_exporter/releases/tag/v1.8.2)                                | v1.6.1                       |
+| `prometheus-adapter`  | [`v0.12.0`](https://github.com/kubernetes-sigs/prometheus-adapter/releases/tag/v0.12.0)                    | v0.11.1                      |
+| `prometheus-operator` | [`v0.76.2`](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.76.2)               | v0.67.1                      |
+| `prometheus`          | [`v2.54.1`](https://github.com/prometheus/prometheus/releases/tag/v2.54.1)                                 | v0.46.0                      |
+| `thanos`              | **DEPRECATED**                                                                                             | v0.34.0                      |
+| `x509-exporter`       | [`v3.17.0`](https://github.com/enix/x509-certificate-exporter/releases/tag/v3.17.0)                        | v3.12.0                      |
+| `karma`               | [`v0.113`](https://github.com/prymitive/karma/releases/tag/v0.113)                                         | No Update                    |
+| `mimir`               | [`v2.14.0`](https://github.com/grafana/mimir/releases/tag/mimir-2.14.0)                                    | v2.11.1                      |
+| `minio-ha`            | [`RELEASE.2024-10-13T13-34-11Z`](https://github.com/minio/minio/releases/tag/RELEASE.2024-10-13T13-34-11Z) | RELEASE.2024-02-09T21-25-16Z |
 
 > Please refer the individual release notes to get a detailed info on the releases.
 
 ## Features 💥
 
-### HAproxy Alerting
+### HAProxy Alerting
 
-Remove HaproxyHttpSlowingDown alert rule
+- Removed `HaproxyHttpSlowingDown` alert rule, it caused false positives adding only noise.
 
 ### prometheus-operated
 
 - Introduced `scrapeConfigNamespaceSelector: {}` and `scrapeConfigSelector: {}` allowing to read all the `scrapeconfig` resources defined in the cluster without any specific annotation.
 
+## Breaking Changes 💔
```

Comment:
@ralgozino 

> Is this a deprecation notice or a breaking change? I’m not sure if the cluster will have errors if it keeps Thanos. In that case, should we explain how to remove Thanos, or at least clarify what the replacement could be?

Reply:
You are right, it is better to call this a Deprecation warning and not a breaking change. I'll change the title.

The replacement is Mimir, as explained in the lines below.